projects/bms_carrier/test/test_passive_balance.c

```diff
@@ -20,99 +20,103 @@ StatusCode TEST_MOCK(ltc_afe_toggle_cell_discharge)(LtcAfeStorage *afe, uint16_t
 // Number of cell voltages to be directly modified during testing
 static const uint8_t NUM_TEST_VOLTAGES = 4;
 
-static LtcAfeStorage s_test_afe_storage;
+// To store voltages for test
+static uint16_t s_test_voltages[NUM_TOTAL_CELLS];
 
-static const uint16_t AFE_DEFAULT_DISCHARGE_BITSET = 0;
+static LtcAfeStorage s_test_afe_storage;
 
-void setup_test(void) {}
+void setup_test(void) {
+  for (uint8_t i = NUM_TEST_VOLTAGES; i < NUM_TOTAL_CELLS; i++) {
+    s_test_voltages[i] = 1010;
+  }
```

Comment:
Should probably initialize the whole array? Maybe set the defaults for the first 4 here, and then change the value of the balance cell in each test